### PR TITLE
Update Horns of War Final Meteor Logic

### PR DESCRIPTION
**Pull request by Skold177**
Improves the logic of final meteor to avoid edge cases that may result in an invulnerable mob. Also changed the duration of stun following Xavers' guidelines.

**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Improves the logic of final meteor to avoid edge cases that may result in an invulnerable mob. Also changed the duration of stun following Xavers' guidelines.

## Steps to test these changes

Pull down these changes and test Horns of War - Chlevnik should now die no matter what, even if its target is not in range.

### CODE REVIEW

File: scripts/zones/Horlais_Peak/mobs/Chlevnik.lua [hunk 1]
```diff
@@ -18,61 +18,51 @@ end
...
     end
 end
 
 entity.onAdditionalEffect = function(mob, target, damage)
-    return xi.mob.onAddEffect(mob, target, damage, xi.mob.ae.STUN, { chance = 25, duration = math.random(5, 10) }) -- 25% chance to stun for 5-10 seconds.
+    return xi.mob.onAddEffect(mob, target, damage, xi.mob.ae.STUN, { chance = 25, duration = 10 }) -- 25% chance to stun for 10 seconds.
 end
 
 entity.onMobWeaponSkill = function(target, mob, skill)
     if skill:getID() == xi.mobSkill.FINAL_METEOR then
```
Comment (WinterSolstice8) [inline scripts/zones/Horlais_Peak/mobs/Chlevnik.lua]: Move this logic into a listener added in onMobInitialize, onMobWeaponskill doesn't fire if you're out of range

```lua
   mob:addListener('WEAPONSKILL_STATE_EXIT', 'FINAL_METEOR_DEATH', function(mobArg, skillID)
        if skillID == xi.mobSkill.FINAL_METEOR then
            mobArg:timer(7000, function(mobArgTimer)
                mobArgTimer:setUnkillable(false)
                mobArgTimer:setHP(0)
            end)
        end
    end
```

**Comment by Skold177** [comment]
ready for review

File: scripts/zones/Horlais_Peak/mobs/Chlevnik.lua [hunk 2]
```diff
@@ -13,70 +13,59 @@ entity.onMobInitialize = function(mob)
...
     end
 
-    if mob:getHP() == 1 and mob:getLocalVar('Meteor') == 0 then
+    if mob:getHP() == 1 and mob:getLocalVar('finalMeteor') == 0 then
         mob:setAutoAttackEnabled(false)
         mob:setMagicCastingEnabled(false)
         mob:setMobAbilityEnabled(false)
-        mob:setLocalVar('Meteor', 1)
+        mob:setMobMod(xi.mobMod.NO_MOVE, 1)
+        mob:setAnimationSub(1)
```
Comment (Xaver-DaRed) [inline scripts/zones/Horlais_Peak/mobs/Chlevnik.lua]: Doesnt this animation make it fall?
It should try to use the skill and then fall, right?

File: scripts/zones/Horlais_Peak/mobs/Chlevnik.lua [hunk 3]
```diff
@@ -13,70 +13,59 @@ entity.onMobInitialize = function(mob)
...
 
-    if mob:getLocalVar('Meteor') == 1 then
-        if mob:checkDistance(target) > 40 then
-            mob:resetEnmity(target)
-        else
-            mob:useMobAbility(xi.mobSkill.FINAL_METEOR)
-        end
+    if mob:getLocalVar('finalMeteor') == 1 then
+        mob:useMobAbility(xi.mobSkill.FINAL_METEOR, nil, nil, true) -- Ignoring distance based off retail capture.
+        mob:setLocalVar('finalMeteor', 2)
```
Comment (Xaver-DaRed) [inline scripts/zones/Horlais_Peak/mobs/Chlevnik.lua:65]: If thats the case, you could just move it here, right in between this 2

**Review by Xaver-DaRed — APPROVED** [review]

**Review by WinterSolstice8 — APPROVED** [review]

File: scripts/actions/mobskills/meteor.lua [hunk 4]
```diff
@@ -11,13 +11,13 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
...
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    local damage = mob:getMainLvl() * 16
+    local damage = mob:getMainLvl() * 32
 
     damage = xi.mobskills.mobMagicalMove(mob, target, skill, damage, xi.element.NONE, 1, xi.mobskills.magicalTpBonus.NO_EFFECT)
     damage = xi.mobskills.mobFinalAdjustments(damage, mob, skill, target, xi.attackType.MAGICAL, xi.damageType.NONE, xi.mobskills.shadowBehavior.IGNORE_SHADOWS)
 
     target:takeDamage(damage, mob, xi.attackType.MAGICAL, xi.damageType.NONE)
-
+    mob:setAnimationSub(1)
```
Comment (MowFord) [inline scripts/actions/mobskills/meteor.lua:20]: `skill:setFinalAnimationSub (1)` the binding was made to be able to set animation subs _after_ the mob skill completes
